File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/RangeOfMotionMeasurementFormViewModel.cs
//$74CE491B
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class RangeOfMotionMeasurementFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class RangeOfMotionMeasurementFormViewModel : BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var rangeOfMotionMeasurementFormService = new RangeOfMotionMeasurementFormServiceController();
            rangeOfMotionMeasurementFormService.RangeOfMotionMeasurementFormInternal(this, objectContext);
            return (RangeOfMotionMeasurementForm)objectContext.GetObject(this._RangeOfMotionMeasurementForm.ObjectID, "RangeOfMotionMeasurementForm");
        }
    }
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/RangeOfMotionMeasurementFormViewModel.ts
//$0C09CC0B
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { RangeOfMotionMeasurementForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class RangeOfMotionMeasurementFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.RangeOfMotionMeasurementFormViewModel, Core';
    @Type(() => RangeOfMotionMeasurementForm)
    public _RangeOfMotionMeasurementForm: RangeOfMotionMeasurementForm = new RangeOfMotionMeasurementForm();
}
